changelog.md

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Is this the version you have ready?....You should add in the compatiblity with the Venafi version.  Maybe tie our version of the connector to the Venafi version?...i.e. 21.3 perhaps?

Reply:
yes, 0.9.6 is the version that's going to master, but I don't get what you mean here!

Reply:
Where did you get 21.3? I can see the Swagger in the VM has version 20.4...

Reply:
Oh sorry, I thought their release numbers followed the current calendar year....go for 20.4 then?